i18n/en-US.json

```diff
@@ -229,6 +229,8 @@
     "REPLUGGED_SETTINGS_DEV_COMPANION_DESC": "Reconnects the Dev Companion coremod to the VSCode extension.",
     "REPLUGGED_SETTINGS_DEV_COMPANION_RECONNECT": "Reconnect",
     "REPLUGGED_SETTINGS_ADVANCED": "Advanced Settings",
+    "REPLUGGED_SETTINGS_AUTOMATIC_RECOVERY":  "Automatic Recovery",
```

Comment:
The strings must not be added manually in the json file, but we do it ourselves with Weblate, which will then update the json.